docker/Dockerfile

```diff
@@ -31,32 +19,26 @@ RUN \
 # But in our setup, it's sorted out by ide-probe instead (org.virtuslab.ideprobe.ide.intellij.IntellijPrivacyPolicy).
 
 # Tools necessary to run non-headless UI tests in the screen-less environment of CI
-RUN \
-  apt-get update; \
-  apt-get install --no-install-recommends -y libxtst6 xvfb xauth x11-apps imagemagick zip; \
-  `# libraries needed by JCEF` \
-  apt-get install --no-install-recommends -y libatk1.0-0 libatk-bridge2.0-0 libgbm1 libglib2.0 libxcomposite1 libxkbcommon0 libxrandr2; \
-  rm -rf /var/lib/apt/lists/*
+RUN apt-get install --no-install-recommends -y libxi6 libxtst6 x11-apps xauth xvfb
+
+# Tools necessary to run manipulate screenshots from ide-probe
+RUN apt-get install --no-install-recommends -y imagemagick zip
 
 # Secondary packages needed in just one (or few) steps of the pipeline:
 # (package       => needed for command(s))
 # binutils       => strings
 # jq             => jq
-# nodejs         => npm (Docker build only), remark
-# procps         => kill (as a standalone command and not shell built-in, to be executed by ide-probe)
 # sudo           => sudo
-# xxd            => xxd
 # unzip          => zipinfo
-RUN \
-  apt-get update; \
-  curl -fsSL https://deb.nodesource.com/setup_16.x | bash -; \
-  apt-get install --no-install-recommends -y binutils jq nodejs procps sudo xxd unzip; \
-  rm -rf /var/lib/apt/lists/*
+# xxd            => xxd
+RUN apt-get install --no-install-recommends -y binutils jq sudo unzip xxd
 
 # Markdown validation utilities
 RUN \
-  npm install --location=global remark-cli remark-lint-no-dead-urls remark-validate-links; \
-  npm uninstall --location=global npm
+  curl -sL https://deb.nodesource.com/setup_18.x | bash -; \
+  apt-get install -y nodejs; \
+  npm install --global remark-cli remark-lint-no-dead-urls remark-validate-links; \
+  remark --help
```

Comment:
we are calling a `remark --help` command here, am I right? for what reason is that?

Reply:
Oh! I had a hard time getting this section to work... mostly using this call to `remark` as an instant "unit test" whether it installed successfully ;) I'd leave it here for good, for that purpose